frontend/src/components/EmployeeModal/EmployeeInfo.tsx

```diff
@@ -29,6 +29,7 @@ const EmployeeInfo = ({
           type="text"
           defaultValue={firstName}
           className={cn(styles.input)}
+          aria-required={true}
```

Comment:
Looks good to me! Looks like you covered all the inputs we need covered. 

A light suggestion: